fusor-ember-cli/app/components/base-f.js

```diff
@@ -16,11 +16,11 @@ export default Ember.Component.extend({
   },
 
   labelClassSize: Ember.computed(function () {
-    return this.getWithDefault('labelSize', 'col-lg-2 col-md-3 col-sm-5');
+    return this.getWithDefault('labelSize', 'col-lg-3 col-md-3 col-sm-4 col-xs-4 col-xs-offset-1');
```

Comment:
@cfchase, I just checked and the original label `col-sm-5` and `input col-sm-7` doesn't look good. I would need to create another offset for `sm`. However, `sm-4` and `sm-5` works good.